src/Swisschain.Sdk.Server/Grpc/Streaming/StreamServiceBase.cs

```
@@ -198,7 +198,16 @@ private async Task RemoveStream(StreamData<TStreamItemCollection, TStreamItem, T
         {
             streamData.CompletionTask.TrySetResult(1);
 
-            _readerWriterLock.AcquireWriterLock(Timeout.Infinite);
+            try
+            {
+                _readerWriterLock.AcquireWriterLock(TimeSpan.FromSeconds(10));
+            }
+            catch (Exception e)
+            {
+                _logger.LogDebug(
```

Why an exception is logged as debug? Are you sure that all exceptions should be swallowed?

---

for now yes, I want to establish the root of error

---

Release beta then